pmpro-gift-levels.php

```diff
@@ -262,7 +262,7 @@ function pmprogl_pmpro_after_checkout($user_id, $morder)
 	global $pmprogl_gift_levels, $wpdb, $pmprogl_existing_member_flag;
 	
 	//which level purchased
-	$level_id = intval($_REQUEST['level']);	
+	$level_id = intval($morder->membership_id);	
```

Comment:
this should also be fetched from $morder if $_REQUEST['level'] is not available

Reply:
Here is the current version of that function in the `dev` branch: https://github.com/strangerstudios/pmpro-gift-levels/blob/790052f7a4210997043a2fb79dc84bfb070d0929/includes/checkout.php#L128

We are now always pulling the level from the order, regardless of if the level is available in `$_REQUEST`. Can you please confirm that the current code looks ok?